tests/integration/components/moderation-list-row/component-test.js

```diff
@@ -25,30 +25,38 @@ test('it renders moderation-list-row accepted with actions', function(assert) {
             }),
         ],
         reviewsState: 'accepted',
-        node: EmberObject.create({ contributors: ['Viper', 'Oogway'] }),
+        node: EmberObject.create({
+            contributors: [{ users: { fullName: 'Viper' } }, { users: { fullName: 'Oogway' } }],
+        }),
     });
     this.render(hbs`{{moderation-list-row submission=submission}}`);
     assert.ok(this.$('[data-status=accepted]').length);
     assert.notOk(this.$('[data-status=pending]').length);
     assert.notOk(this.$('[data-status=rejected]').length);
 
-    assert.equal(this.$('[data-status=accepted]').text().trim(), 'accepted on October 27, 2017 by Kung-fu Panda');
+    assert.equal(this.$('[data-status=accepted]').text().replace(/\s+/g, ' ').trim(), 'Submitted 2 months ago by' +
```

Comment:
Does this test depend on the current date being about 2 months after October 27?